structuredefinitions/UKCore-Observation-InspiredOxygen.xml

```diff
@@ -68,5 +68,20 @@
       <path value="Observation.value[x].code" />
       <fixedCode value="l/min" />
     </element>
+     <element id="Observation.method">
```

Comment:
This is not needed - it should be brought in from UKCore-Obserrvation

Reply:
Done this change